app/src/main/java/org/oppia/android/app/player/state/DragDropSortInteractionView.kt

```diff
@@ -45,10 +51,68 @@ class DragDropSortInteractionView @JvmOverloads constructor(
   private lateinit var dataList: List<DragDropInteractionContentViewModel>
   private lateinit var onDragEnd: OnDragEndedListener
   private lateinit var onItemDrag: OnItemDragListener
+  private var itemTouchHelper: ItemTouchHelper? = null
+  companion object {
+    private const val LONG_PRESS_TIMEOUT_MS = 30L
```

Comment:
This is extremely short for a long press. Users might not even notice it. Consider increasing it to up to 300ms.